samples/subsys/task_wdt/sample.yaml

```diff
@@ -0,0 +1,18 @@
+sample:
+  name: Task Watchdog Subsytem Sample
+tests:
+  sample.subsys.task_wdt:
+    tags: subsys
+    harness: console
+    harness_config:
+        type: multi_line
+        ordered: true
+        regex:
+            - "Task watchdog sample application."
+            - "Control thread started."
+            - "Main thread still alive..."
+            - "Control thread getting stuck..."
+            - "Task watchdog callback (channel: 1, thread: control)"
+            - "Resetting device..."
+            - "Task watchdog sample application."
+    depends_on: watchdog task_wdt
```

Comment:
Is this depends line correct?

Reply:
Good spot. It should actually be possible to run this sample with any board (even if it doesn't have a hardware watchdog). So I'll remove the dependency.

Reply:
FYI: I finally replaced this line with
```
    depends_on: watchdog
    integration_platforms:
      - nucleo_g474re
```
To limit the number of CI runs to only one and not compile it for almost every board.